src/app/inventory/store/stats.ts

```diff
@@ -401,38 +401,46 @@ function attachPlugStats(
   // The plug that is currently inserted into the socket
   const activePlug = socket.plugged;
 
-  // We need to calculate the base investment stat value for the item so we can correctly
-  // interpolate each plug's effect on the item.
+  /** This holds the item's 'base' investment stat values without any plug additions. */
   const baseItemInvestmentStats: { [statHash: number]: number } = {};
 
+  /*
+   * The active plug is already contributing to the items stats in statsByHash. Thus we treat it seperately
+   * here for two reasons,
+   * 1. we need to calculate the 'base' investment stat value for the item's stats so that we can calculate
+   * correct values for the inactive plugs.
+   * 2. By utilising the fact that the items stats already include this, we can do one less interpolation
+   * per stat to figure out the plugs stat contribution.
+   */
```

Comment:
```suggestion
  /*
   * The active plug is already contributing to the item's stats in statsByHash. Thus we treat it separately
   * here for two reasons,
   * 1. we need to calculate the 'base' investment stat value (without this plug's contribut
   ) for the item's stats so that we can calculate
   * correct values for the inactive plugs.
   * 2. By utilizing the fact that the item's stats already include this, we can do one less interpolation
   * per stat to figure out the active plug's stat contribution.
   */
```